packages/nimble-components/src/combobox/index.ts

```diff
@@ -70,6 +70,26 @@ export class Combobox
     @observable
     public controlWrapper?: HTMLElement;
 
+    public override get value(): string {
+        return super.value;
+    }
+
+    // This override is to work around an issue in FAST where an old filter value
+    // is used after programmatically setting the value property.
+    // See: https://github.com/microsoft/fast/issues/6749
+    public override set value(next: string) {
+        super.value = next;
+        if (!this.valueUpdatedByInput) {
+            // Using index notation to avoid error from accessing private member
```

Comment:
Just emphasize that it's a workaround and tied to a specific issue

```suggestion
            // Workaround using index notation to manipulate private member
            // Can remove when following resolved: https://github.com/microsoft/fast/issues/6749
```